cfme/tests/configure/test_access_control.py

```diff
@@ -104,7 +106,7 @@ def test_user_duplicate_name():
 @pytest.mark.tier(3)
 def test_username_required_error_validation():
     user = User(
-        name=None,
+        name="",
```

Comment:
actually,  this change is not necessary but from my perspective empty "name" input field should be empty string not None.
 I can remove this change if you want